components/list/test/list.vdiff.js

```diff
@@ -284,20 +302,6 @@ describe('list', () => {
 		});
 	});
 
-	describe('illustration', () => {
```

Comment:
Previously we were using a `div` for testing this, which isn't really what we expect or support. This case is now better tested in the new list and tiles tests. There are a few more like this, which will eventually be removed as the tests are cleaned up.